rootfs-builder/rootfs.sh

```diff
@@ -385,7 +396,30 @@ if [ -z "${AGENT_SOURCE_BIN}" ] ; then
 	[ -n "${AGENT_VERSION}" ] && git checkout "${AGENT_VERSION}" && OK "git checkout successful"
 	make clean
 	make INIT=${AGENT_INIT}
-	make install DESTDIR="${ROOTFS_DIR}" INIT=${AGENT_INIT} SECCOMP=${SECCOMP}
+	make install DESTDIR="${ROOTFS_DIR}" INIT=${AGENT_INIT} SECCOMP=${SECCOMP} \
+		TRACE="${AGENT_TRACE}" TRACE_DEV_MODE="${TRACE_DEV_MODE}"
+
+	# List of additional agent systemd services (from the agent repository)
+	services=()
+
+	[ "${AGENT_TRACE}" = "yes" ] && services+=("jaeger-client-socat-redirector.service")
```

Comment:
@jodh-intel where is the jaeger-client-socat-redirector.service file? Is is already present in the rootfs?